gems/aws-sdk-core/lib/aws-sdk-core/plugins/stub_responses.rb

```diff
@@ -46,8 +54,20 @@ def after_initialize(client)
         end
       end
 
-      class Handler < Seahorse::Client::Handler
+      class ApiRequestsHandler < Seahorse::Client::Handler
```

Comment:
Logs the api request (the anonymous handle in setup_stubbing) but prior to sending so presigned urls are still tracked.